### DISCUSSION TITLE
2023 - Zanvoort Circuit - Plot

### DISCUSSION THREAD

**Discussion by adamhott** [br-g/openf1#59] [Q&A]
[zandvoort_data.csv](https://github.com/br-g/openf1/files/15050999/zandvoort_data.csv)

Hello, I'm trying to plot Max Verstappen's 2023 Grand Prix race at Zandvoort. I've successfully pulled down the x, y, ,and z data into a .csv file and have a Rust program to plot the data, but the result doesn't look like the actual Zandvoort track so I'm wondering what adjustments I need to make.

![Zandvoort_Circuit](https://github.com/br-g/openf1/assets/24814301/f37aa818-b28d-4b7f-b23e-19b6240c7570)
![zandvoort_track_adjusted](https://github.com/br-g/openf1/assets/24814301/2e0eb852-a211-4afa-9cff-2a6ce250a326)

Attached is the data file, the plot, and, what the actual track should look like.

Here is my repo with Rust code:
https://github.com/CodingInGreen/f1-led-circuit-track-plot

Any ideas are welcome! Thanks!

**Comment by adityakotha03** [comment]
Hi, I am not sure why this happened, we are currently building something similar in React here https://www.f1nsight.com/, this is our code https://github.com/adityakotha03/F1nsight. We haven't made the map for Zanvoort Circuit yet, but may be our models might help you out. When we build the map and test it out I will let you know if the same issues persists with us.

**Reply by adamhott** [reply]
Thank you so much! This seems to be very helpful, I'll take a look and let me knwo when you build out Zanvoort!

**Comment by adamhott** [comment]
I was able to solve this, I was actually pulling in the wrong dataset, a dataset for the wrong circuit!

**Reply by adityakotha03** [reply]
I was goona tell you abt this too, because I was going through the data a couple of days ago to get the coordinates and found it was matching with the map. Anyways cheers for solving it.
